data/talkactions/scripts/buyhouse.lua

```diff
@@ -1,10 +1,20 @@
+local config = {
+	level = 1,
+	onlyPremium = true
+}
+
 function onSay(player, words, param)
 	local housePrice = configManager.getNumber(configKeys.HOUSE_PRICE)
 	if housePrice == -1 then
 		return true
 	end
 
-	if not player:isPremium() then
+	if player:getLevel() < config.level then
+		player:sendCancelMessage("You need level " .. config.level .. " or higher to buy a house.")
+		return false
+	end
+
+	if not player:isPremium() and config.onlyPremium then
```

Comment:
This is fine, but if you first check `config.onlyPremium` it would be the correct way to write it, since once it is false, `isPremium` will never be called because it will never be necessary